joomla/joomla-cms
# [4.0] Namespace com_modules

Pull Request for Issue # .

### Summary of Changes
As title says, this PR convert com_modules component to namespace MVC for Joomla 4.

### Testing Instructions
Overall, test and make sure it works in the none namespace version of the extension

1. Install a copy of my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_modules.zip (to avoid problem with patchtesters component not apply patch properly)

2. Login to administrator area, access Extensions -> Modules, try to add/edit/publish/unpublish/checkin ...modules. Make sure it is working as expected.

3. Access to frontend of the site, login using a super admin account, try to edit a module, save it, make sure it is working as expected. That's the only function could be tested right now for frontend as edit article feature in frontend right now is broken, so we cannot test insert module using editor-xtd button.
